src/backend/columnar/sql/columnar--9.5-1--10.0-1.sql

```diff
@@ -3,6 +3,8 @@
 CREATE SCHEMA cstore;
 SET search_path TO cstore;
 
+CREATE SEQUENCE cstore_storageid_seq NO CYCLE;
```

Comment:
the name of the sequence reads `cstore.cstore_storageid_seq`. Would you mind removing the `cstore` clasifier from the sequence as long as it lives in the `cstore` schema?

I think this is part of a bigger discussion on naming. Given we want to rename everything to columnar. It is undecided what schema we will put our objects in though.